etc/kayobe/kolla/globals.yml

```diff
@@ -22,6 +22,12 @@ kolla_image_tags:
 # Variables defining which tag to use for each container's image.
 {{ lookup('pipe', 'python3 ' ~ kayobe_config_path ~ '/../../tools/kolla-images.py list-tag-vars') }}
 
+# NOTE(priteau): Nova container images can use Rocky Linux 9.3 to avoid
```

Comment:
nit: I would have put this in kolla-image-tags.yml, something like this:
```
kolla_image_tags:
  nova:
    rocky-9: "{% if stackhpc_pulp_repo_rocky_9_minor_version < 4 %}2023.1-rocky-9-20240725T200915{% else %}2023.1-rocky-9-20240723T140948{% endif %}
```

Reply:
I tried this first…

```
TASK [kolla-ansible : Ensure the Kolla global configuration file exists] *******
Monday 05 August 2024  15:44:09 +0100 (0:00:00.293)       0:00:03.777 ********* 
fatal: [localhost]: FAILED! => 
  msg: 'An unhandled exception occurred while running the lookup plugin ''pipe''. Error was a <class ''ansible.errors.AnsibleError''>, original message: lookup_plugin.pipe(python3 /home/stack/pierre/src/kayobe-config/etc/kayobe/../../tools/kolla-images.py list-tag-vars) returned 1. lookup_plugin.pipe(python3 /home/stack/pierre/src/kayobe-config/etc/kayobe/../../tools/kolla-images.py list-tag-vars) returned 1'
```

Reply:
:(

Reply:
I didn't want to waste more time on this issue as it is only useful for a few environments and could be dropped when Rocky Linux 9.4 images are confirmed to be working fine.

Reply:
That's fine. The kolla-images.py script expects the file to be plain YAML.